code/datums/outfit.dm

```diff
@@ -220,6 +221,9 @@
 	if(socks)
 		user.socks = initial(socks.name)
 
+	if(bra)
+		user.bra = initial(bra.name)
```

Comment:
Add modular edit comments around this for a multi-line edit, which is one with a START and one with an END, you can see examples elsewhere in the code you touched.